### ISSUE TITLE
add ng-conf announcement to the homepage

### ISSUE THREAD

**Issue by IgorMinar** [angular/angular#22122]
https://www.ng-conf.org/
April 18–20th 2018 Salt Lake City, UT
campaign duration: now - April 21th

logo: provided in https://github.com/angular/angular/issues/22029#issuecomment-363631691 ([but it doesn't fit our design](https://github.com/angular/angular/issues/22029#issuecomment-363717345) - @petebacondarwin can you clarify the problem?)

@aaronfrost can provide a better logo

previous logo for ng-atl, angular mix or AC logos found here: https://github.com/angular/angular/tree/master/aio/content/images/marketing/home

please note that https://github.com/angular/angular/pull/22043 is landing and it will change how this change should be implemented.

Ref https://github.com/angular/angular/issues/22029#issuecomment-363631691 by @StephenFluin

**Comment by aaronfrost** [comment]
I am on this. Thanks @IgorMinar for the example. I will get this today.

**Comment by petebacondarwin** [comment]
The problem is that the logo that is given is great for a large display but in this banner, it will be quite small and it is too fiddly and not clear enough. See

![image](https://user-images.githubusercontent.com/15655/36046952-ee33e26a-0dda-11e8-95aa-fb2e02670d1a.png)

**Comment by aaronfrost** [comment]
Would this file work? @petebacondarwin 
https://drive.google.com/file/d/1-TFOeqpMGhhSsNbojCswlzIjf5C5FtdQ/view?usp=sharing

![dark logo square](https://user-images.githubusercontent.com/662832/36055243-4167a0ac-0db8-11e8-80ea-2b1f990d635b.jpg)

**Comment by StephenFluin** [comment]
I'm worried we're headed in the wrong direction. I think we need fewer colors instead of more. Like the monochrome one in the favicon: https://www.ng-conf.org/wp-content/themes/ng-conf/favicon.ico

**Comment by StephenFluin** [comment]
![image](https://user-images.githubusercontent.com/165056/36055559-2aab5fb2-0db2-11e8-88d3-d897ed9f9820.png)
Just made this by coloring from the alpha channel of your logo. Thoughts?

**Comment by StephenFluin** [comment]
Here's another one made by cropping the gradient: 
![image](https://user-images.githubusercontent.com/165056/36055588-64ffd5c6-0db2-11e8-97da-f7bfbcdd2028.png)

**Comment by petebacondarwin** [comment]
I like the monochrome one. Remember this logo is going to be really small in the banner

**Comment by IgorMinar** [comment]
Yup. Monochrome is better for us but Frosty should be OK with what we end
up using. It's their brand. They should provide us with a monochrome logo
or ok Stephen's mod.

**Comment by aaronfrost** [comment]
The black and white one is good enough for now. We are more interested in getting this up on your site and getting the publicity than we are about getting a specific image. 

Thanks all for working on this.

**Comment by aaronfrost** [comment]
This one is more modern and it's b&w https://drive.google.com/file/d/0B0poyiQbU1uqVTNfc3pkNDBoR0tYZEpHRDBfNUZCcVBDVE84/view?usp=drivesdk

**Comment by IgorMinar** [comment]
Can you send us a pr? You just need to
edit aio/content/marketing/announcements.json and add image to
https://github.com/angular/angular/tree/master/aio/content/images/marketing/home
.

Please resize the image to 60px height.

Use text and dates from this issue for the json fields. Thanks.

**Comment by petebacondarwin** [comment]
If that's your brand it works for me!
